README.md

```diff
@@ -105,6 +107,7 @@ nmap <unique> <leader>pt <Plug>PickerTabedit
 nmap <unique> <leader>pv <Plug>PickerVsplit
 nmap <unique> <leader>pb <Plug>PickerBuffer
 nmap <unique> <leader>p] <Plug>PickerTag
+nmap <unique> <leader>pb] <Plug>PickerBufferTag
```

Comment:
That is perfect! I use a different mapping myself, so I just smooshed `pb` and `p]` together.